src/data/content/design-system/componenti/card.yaml

```diff
@@ -120,7 +188,34 @@ tabs:
             title: Anatomia
             headingLevel: 2
             text: |
-              _In stesura_
+              La maggior parte delle card è composta dagli stessi elementi, con poche eccezioni:
+
+              1. **sfondo**: la card ha un colore, solitamente bianco, e un'ombra che la mette in evidenza;
+              2. **titolo**: si tratta del nome del contenuto collegato dalla card;
+              3. **sommario**: una sintesi del contenuto della card;
+              4. **immagine**: è possibile utilizzare diversi formati - vedi proporzioni media;
+              5. **etichetta**: comunemente mostra la categoria. Può fungere da titolo o da link;
+              6. **data**: indica quando è stato pubblicato il contenuto. Non sempre è necessario;
+              7. **link**: un link "leggi tutto" può essere inserito per stimolare l'utente a scoprire di più. Non sempre è necessario. 
+              8. **bordo**: è presente solo nella variante "Highlight", mette in risalto la card;
+
+              > *Attenzione: le varianti presenti in questa scheda non sono disponibili nella libreria Bootstrap Italia.*
```

Comment:
Non capisco questo warning.

Reply:
Stiamo dettagliando varianti non disponibili in sviluppo. Il potenziale utente del DS che si guarda l'anteprima di componenti e poi le specifiche, potrebbe non capire perché vede cose diverse da una parte e dall'altra. 

Mettere un disclaimer nei componenti che prevedono questa problematica mi sembrava un atto di chiarezza. Possiamo anche spiegare il perché di questa situazione.

Reply:
Apriamo issue ad hoc sia su repo ui kit che bsi per aggiornare il componente e le sue varianti, anche alle evoluzioni necessarie lato accessibilità. Ma documentiamo solo cose di cui siamo sicuri. La manutenibilità nel tempo di queste pagine sennò diventa impossibile. È difficile ricordarsi dove stanno note così tra qualche tempo, diventerebbero solo motivo di frammentazione.

Reply:
Ho aperto intanto una issue sulla repo UI Kit Italia qui per tracciarlo: https://github.com/italia/design-ui-kit/issues/115. Faccio altrettanto su Bootstrap Italia a breve. Per ora togli pure la nota e lascia la descrizione delle varianti in UI Kit Italia così com'è direi per semplificare. Tra la beta e la release ufficiale cercheremo di prioritizzare la risoluzione del problema a monte.